tests/test_executor.py

```diff
@@ -768,6 +768,31 @@ def test_group_by(self):
                 self.assertEqual(result.columns, columns)
                 self.assertEqual(result.rows, expected)
 
+    def test_array_unique_agg(self):
```

Comment:
I didn't really like the idea of writing a test for one function, although it didn't seem like it should be grouped with the other functions as the test data didn't seem appropriate for unique testing

Reply:
What about adding it to `test_group_by`?